admin/herbert_mex_mpi.m

```diff
@@ -2,9 +2,10 @@
 % the script to build cpp communicator using Matlab compiler and not using
 % cmake.
 %
-% one needs to manually modify this script to specify the mpi libraries
-% location in the system.
+% Manually modify this script to specify the mpi libraries location in your
```

Comment:
Does this want to become a CMake configuration file, as CMake knows where MPI is.

Reply:
This file has been created well before cmake and is intended for people who does not have/know cmake and want to build it. Its nice to have two ways of doing something.